INN/umbrella-currentorg
# Create category specific tags

## Changes

This pull request makes the following changes:

- Adds the ability to choose a parent category for tags in the wpbdp_tag taxonomoy. On the listing edit screen, tags will only display when their parent category is selected.
## Why



For #23 as part of the [Current Business Directory](https://github.com/INN/umbrella-currentorg/projects/1) project.

## Testing/Questions

Features that this PR affects:

- Categories and tags from the Business Directory plugin.

Steps to test this PR:

1. Create/edit existing Business Directory plugin tag and give it a parent category.
2. On the listing edit page, select a parent category and make sure its child tags appear in the `Directory Tags` metabox.
3. On the listing edit page, remove a parent category and make sure its child tags are also removed in the `Directory Tags` metabox.
